rest-service/manager_rest/config.py

```diff
@@ -50,6 +50,7 @@ def __init__(self):
         self._security_authentication_providers = []
         self._security_authorization_provider = None
         self._insecure_endpoints_disabled = False
+        self._security_agent_rest_credentials = None
```

Comment:
OK. Same for the node property? https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/365/files#diff-9a67a043795bd3da7b6364194f907257R157